fuel-crypto/benches/signature.rs

```diff
@@ -251,10 +247,14 @@ fn signatures(c: &mut Criterion) {
 
     group_recover.bench_with_input(
         "k256",
-        &(k2_recoverable, k2_digest),
-        |b, (recoverable, digest)| {
+        &(k2_signature, k2_recovery_id, k2_digest),
+        |b, (signature, recovery_id, digest)| {
             b.iter(|| {
-                recoverable.recover_verifying_key_from_digest(black_box(digest.clone()))
+                k256::ecdsa::VerifyingKey::recover_from_digest(
```

Comment:
Shouldn't we be benching off of the library we use during regular block execution in std?

Reply:
Also why remove the black box?

Reply:
We bench from both `k256` and `secp256k1` libraries, since we want both of the benchmarks.

Reply:
The inputs are automatically passed through a `black_box` [docs](https://bheisler.github.io/criterion.rs/book/user_guide/benchmarking_with_inputs.html#benchmarking-with-one-input). `black_box`ing the result of `.clone()` adds overhead that's not part of what we're benchmarking here.